src/model_desc/comment_desc.ml

```diff
@@ -115,12 +118,20 @@ and tag : general_tag t =
       (function
       | `Url -> C0 "`Url" | `File -> C0 "`File" | `Document -> C0 "`Document")
   in
+  let raise_kind =
+    (* TODO: any way to avoid duplication of inline_element? variance on GADT doesn't work *)
+    Variant
+      (function
+      | `Code_span x -> C ("`Code_span", x, string)
+      | `Reference (x1, x2) ->
+          C ("`Reference", (x1, x2), Pair (reference, link_content)))
+  in
   Variant
     (function
     | `Author x -> C ("`Author", x, string)
     | `Deprecated x -> C ("`Deprecated", x, docs)
     | `Param (x1, x2) -> C ("`Param", (x1, x2), Pair (string, docs))
-    | `Raise (x1, x2) -> C ("`Raise", (x1, x2), Pair (string, docs))
+    | `Raise (x1, x2) -> C ("`Raise", (x1, x2), Pair (raise_kind, docs))
```

Comment:
You can coerce to the bigger type to avoid writing `raise_kind`:
```suggestion
    | `Raise (x1, x2) -> C ("`Raise", ((x1 :> general_inline_element), x2), Pair (inline_element, docs))
```

Reply:
Thanks for the hint!